examples/guided.py

```diff
@@ -150,48 +186,76 @@ def ask_user_questions():
 		archinstall.arguments['harddrive'].keep_partitions = False
 
 	# Get disk encryption password (or skip if blank)
-	if archinstall.arguments['harddrive'] and archinstall.arguments.get('!encryption-password', None) is None:
-		if (passwd := archinstall.get_password(prompt='Enter disk encryption password (leave blank for no encryption): ')):
+	if (
+		archinstall.arguments['harddrive']
+		and archinstall.arguments.get('!encryption-password', None) is None
+	):
+		if (
+			passwd := archinstall.get_password(
+				prompt='Enter disk encryption password (leave blank for no encryption): '
+			)
+		) :
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability. Absolutely NO! Flashbacks from PHP for some reason.